### ISSUE TITLE
Gnuroot takes wakelocks by default

### ISSUE THREAD

**Issue by tuomas2** [corbinlc/GNURootDebian#56]
Wakelocks should be optional, non-defaul behavior. 

In addition, manual dropping wakelocks dont always work with gnuroot in some cases. I'll add more info here when I know more exactly when this happens.

**Comment by AdamMatthewLTM** [comment]
Hi all,

I am currently assisting the developers in sorting relevant issues for GNURoot Debian on GitHub. If the issue discussed is still relevant, could you please reply to this thread and state the issue needs to be resolved and I will attempt to sort it for you ASAP. If the issue is no longer relevant, please reply to this thread and state it has been resolved, and I will suggest this thread be closed.

Furthermore, if a particular issue is considered to be high priority, please state that you are requesting it to be considered as high priority and I will mark it as such.

Thank you all so much for your time and understanding,
-Adam

**Comment by cme** [comment]
Yes, this is still an issue -- it still takes wakelocks by default, and having a terminal open with an idle command prompt should absolutely not keep the device awake.

**Comment by AdamMatthewLTM** [comment]
@cme , 

Thank you for your reply. I am recommending this issue remain open as medium priority until further notice. 

Thank you for your patience in these matters,
-Adam
